packages/frontend/nginx.conf

```diff
@@ -8,7 +8,7 @@ http {
         index index.html;
 
         location /api/ {
-          proxy_pass http://backend:3000/;
+          proxy_pass ${AP_BACKEND_URL}/;
```

Comment:
How does this environment variable get replaced?

Reply:
That's already built-in inside nginx, anything in template each time the docker run it will substitute and create new configs.